config/locales/errors/tr.yml

```diff
@@ -6,3 +6,4 @@ tr:
     invalid_email: Geçerli bir e-mail adresi değil!
     system_error: Sistemsel bir hata oluştu. Lütfen daha sonra tekrar deneyin.
     unicode_not_supported: Alıcı numara unicode desteklemiyor!
+    invalid_authenticity_token: Geçersiz kimlik doğrulama jetonu! Lütfen tekrar deneyin.
```

Comment:
Neyi deneyin?

Reply:
`authenticity_token` sadece login formunda değil, diğer pek çok formda kullanıldığı için, "Lütfen tekrar giriş yapmayı deneyin" demek istemedim. Bunun yerine genel bir mesaj yazmak istedim.

Alternatif olarak aklıma "Lütfen formu doldurup tekrar deneyin." geliyor.

Reply:
O vakit iki farklı mesaj kullanalım.  Yani "Lütfen tekrar giriş yapmayı deneyin" mesajı için ayrı string açıp alert'te kullanalım.

Reply:
Önerimi geri alıyorum, bu mesaj benzer tüm durumlarda kullanılacağından bölemiyoruz..  Böyle devam edelim.